cfme/tests/cloud/test_providers.py

```diff
@@ -827,6 +827,48 @@ def test_domain_id_validation(request, provider):
     assert view.flash.messages[0].type == 'error'
 
 
+@test_requirements.azure
+@pytest.mark.provider([AzureProvider], override=True, selector=ONE)
+def test_vpc_env_selection(setup_provider, provider, appliance, provisioning):
+    """
+    Test selection of components in environment page of cloud instances
+    with selected virtual private cloud
+
+    Polarion:
+        assignee: anikifor
+        casecomponent: WebUI
+        initialEstimate: 1/2h
+        testSteps:
+            1. Provision an Azure Instance from an Image.
+            2. At the environment page, try to select components without vpc with vpc
```

Comment:
This should be : 
2. At the environment page, try to select components with vpc and without vpc

Reply:
replaced with `2. At the environment page, try to select components with vpc`.

this test should cover only provisioning with vpc, some other provisioning test provisions instances without selecting vpc